src/CarbonAware.DataSources/CarbonAware.DataSources.ElectricityMaps/src/Model/EmissionsFactor.cs

```diff
@@ -0,0 +1,10 @@
+using System.Text.Json.Serialization;
+
+namespace CarbonAware.DataSources.ElectricityMaps.Model;
+
+[JsonConverter(typeof(JsonStringEnumConverter))]
+public enum EmissionsFactor
```

Comment:
🐑 include link to `ElectricityMaps` webpage explaining what these are, not intuitive unless you've seen it in their docs